### PR TITLE
ARROW-3977: [Gandiva] fix label during ctest invoc

### PR DESCRIPTION
**Pull request by pravindra**

### CODE REVIEW

File: ci/travis_script_gandiva_cpp.sh [hunk 1]
```diff
@@ -23,10 +23,7 @@ source $TRAVIS_BUILD_DIR/ci/travis_env_common.sh
 
 pushd $CPP_BUILD_DIR
 
-PATH=$ARROW_BUILD_TYPE:$PATH ctest -j2 --output-on-failure -L gandiva,unittest
-
-# not running in parallel, since some of them are benchmarks
-PATH=$ARROW_BUILD_TYPE:$PATH ctest -VV -L gandiva,integ
+PATH=$ARROW_BUILD_TYPE:$PATH ctest -j2 --output-on-failure -L gandiva
```
Comment (wesm) [inline ci/travis_script_gandiva_cpp.sh]: Do you still want `-VV`?

Comment (wesm) [reply ci/travis_script_gandiva_cpp.sh]: Noted

**Comment by xhochy** [comment]
The tests are actually failing when run inside of valgrind.

**Comment by pravindra** [comment]
The gandiva weren't running inside valgrind till now - the ARROW_TRAVIS_VALGRIND option was being ignored. So, there are tonnes of valgrind errors that need to be debugged/fixed.

I want to disable valgrind for the gandiva tests, and enable them in this jira so that the tests at least run as part of CI. Then, work on the valgrind errors in a different jira.

**Comment by pravindra** [comment]
@wesm can you please merge this change ? I'll fix the valgrind issues as part of ARROW-3979

**Review by wesm — APPROVED** [review]
+1

**Pull request closed without merging**